tos.md

```diff
@@ -0,0 +1,84 @@
+# Network Commons License (v1.0.4)
+
+The Network Commons License is the framework that governs the network commons.
+
+## The License
+
+The Network Commons License (NCL) describes the terms and conditions associated with the use of free and
+open networks, and so enables individuals, communities, companies, governments and other organizations to
+adopt or support the same.
+
+The NCL is rooted in the following four tenets:
+- Participants are free to use the network for any purpose that does not limit the freedom of others
+to do the same.
+- Participants are free to know how the network and its components function.
+- Participants are free to offer and accept services on the network on their own terms.
+- By joining or adding services to the free network, you agree to extend the network to others under
+the same conditions.
+
+## Terms
+
+### Participation
+- The free network honors the human right to communication, and aims for the maximum possible
+benefit for its constituents. Disputes regarding the right and proper execution of this framework
+should be resolved through the consensus of network stakeholders according to these principles.
+- Network devices and infrastructure can be financed and capitalized in any manner that does not
+seek to profit off of other participants in the network.
+- The network as a whole does not have any owner or proprietor, regardless of any specific
+contribution to the network.
+- Network participants agree to extend the network on the same terms and conditions that they
+receive it, allowing all data to transit their own portion of the network without manipulating that
+data for any reason other than network maintenance.
+- Network participants agree only to the terms of this License - any other agreement must be
+explicitly expressed and accepted.
+- In order to maximize connectivity and network growth, network constituents are encouraged to
+allow other network participants to install devices in their facilities, but this permission is always
+discretionary and can be revoked at any time.
+- Commercial access to the network must extend and share the network.
```

Comment:
I'm not sure how this is different from the point above, "Network participants agree to extend the network on the same terms and conditions that they ￼receive it"

Reply:
Agreed. This has been deleted.